model/src/test/kotlin/utils/FindingsMatcherTest.kt

```diff
@@ -84,12 +81,13 @@ class FindingsMatcherTest : WordSpec() {
 
         "getClosestCopyrightStatements()" should {
             "return exactly the statements within the line threshold" {
-                val licenseStartLine = THRESHOLD + 20
+                // Use an arbitrary license start line that is larger than TOLERANCE_LINES_DEFAULT_VALUE.
+                val licenseStartLine = 20
```

Comment:
To protect against changes in the default value it makes sense to generate the value here, like before.

Reply:
Done in a follow-up commit to avoid conflicts during rebase.